src/bug_report.rs

```diff
@@ -17,16 +18,33 @@ pub fn create() {
         starship_config: get_starship_config(),
     };
 
-    let link = make_github_issue_link(environment);
-
-    if open::that(&link).is_ok() {
-        println!("Take a look at your browser. A GitHub issue has been populated with your configuration.");
-        println!("If your browser has failed to open, please click this link:\n");
+    let issue_body = get_github_issue_body(&environment);
+
+    println!(
+        "{}\n{issue_body}\n\n",
+        Style::new().bold().paint("Generated bug report:")
+    );
+    println!("Forward the pre-filled report above to GitHub in your browser?");
+    println!("{} To avoid any sensitive data from being exposed, please review the included information before proceeding. Data forwarded to GitHub is subject to GitHub's privacy policy.", Style::new().bold().paint("Warning:"));
+    println!(
+        "Enter `{}`, or anything else to decline, and `{}` to confirm your choice:\n",
+        Style::new().bold().paint("y"),
+        Style::new().bold().paint("Enter key")
+    );
+
+    let mut input = String::new();
+    let _ = std::io::stdin().read_line(&mut input);
+
+    if input.trim() == "y" {
```

Comment:
Do we want to accept if the user enters `Y`?

```suggestion
    if input.trim().to_lowercase() == "y" {
```